File: AddIns/Templates/VBCMasterDetailReport/Form1.vb
Public Class TAG_FORMNAME
    Inherits Srvtools.InfoForm
    Private Sub TAG_FORMNAME_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        clientQuery1.Show(Me.panel1)
        AddHandler Me.reportViewer1.LocalReport.SubreportProcessing, AddressOf LocalReport_SubreportProcessing
        Me.reportViewer1.LocalReport.DataSources.Add(New Microsoft.Reporting.WinForms.ReportDataSource("NewDataSet_", Me.DataSet.RealDataSet.Tables(0)))
    End Sub

    Private Sub LocalReport_SubreportProcessing(ByVal sender As Object, ByVal e As Microsoft.Reporting.WinForms.SubreportProcessingEventArgs)
        e.DataSources.Add(New Microsoft.Reporting.WinForms.ReportDataSource("NewDataSet_", Detail))
    End Sub

    Private Sub btClear_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btClear.Click
        clientQuery1.Clear(Me.panel1)
    End Sub

    Private Sub btQuery_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btQuery.Click
        clientQuery1.Execute(Me.panel1)
        Me.reportViewer1.RefreshReport()
    End Sub
End Class

File: AddIns/Templates/CEasilyReportSingle1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Srvtools;

namespace TAG_NAMESPACE
{
    public partial class TAG_FORMNAME : InfoForm
    {
        public TAG_FORMNAME()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            clientQuery1.Execute(splitContainer1.Panel1);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            clientQuery1.Clear(splitContainer1.Panel1);
        }

        private void TAG_FORMNAME_Load(object sender, EventArgs e)
        {
            clientQuery1.Show(splitContainer1.Panel1);
        }

        private void btnPrint_Click(object sender, EventArgs e)
        {
            easilyReport1.Execute(false);
        }
     }
}

